scripts/tslint/booleanTriviaRule.ts

```diff
@@ -25,38 +18,37 @@ function walk(ctx: Lint.WalkContext<void>, checker: ts.TypeChecker): void {
     }
 
     function checkCall(node: ts.CallExpression): void {
-        if (!node.arguments || !node.arguments.some(arg => arg.kind === ts.SyntaxKind.TrueKeyword || arg.kind === ts.SyntaxKind.FalseKeyword)) {
-            return;
-        }
-
-        const targetCallSignature = checker.getResolvedSignature(node);
-        if (!targetCallSignature) {
+        if (!node.arguments) {

```

Comment:
Is it possible for `CallExpression.arguments` to ever be undefined?
As long as you don't handle `NewExpression` here you shouldn't need to worry about `undefined`.